src/system/RoutesModule/Twig/TwigExtension.php

```diff
@@ -135,4 +135,32 @@ public function displayPathAsString($path, $route)
 
         return $prefix . '<strong>' . $path . '</strong>';
     }
+
+    public function getObjectState($state = 'initial', $uiFeedback = true)
+    {
+    }
+
+    public function templateHeaders($contentType, $asAttachment = false, $fileName = '')
+    {
+    }
+
+    public function getObjectTypeSelector()
+    {
+    }
+
+    public function getTemplateSelector()
+    {
+    }
+
+    public function getUserAvatar($uid, $width = 0, $height = 0, $size = 0, $rating = '')
+    {
+    }
+
+    public function getImageThumb($params)
+    {
+    }
+
+    public function profileLink($uid, $class = '', $maxLength = 0)
+    {
+    }
```

Comment:
`getUserAvatar` is used here:
https://github.com/zikula/core/blob/1.4/src/system/RoutesModule/Resources/views/Helper/includeStandardFieldsDisplay.html.twig#L18
https://github.com/zikula/core/blob/1.4/src/system/RoutesModule/Resources/views/Helper/includeStandardFieldsDisplay.html.twig#L25
So you might remove that as well.